src/main.js

```diff
@@ -0,0 +1,79 @@
+var saveButton = document.querySelector('#save');
+var titleInput = document.querySelector('#titleInput');
+var bodyInput = document.querySelector('#bodyInput');
+var ideaParent = document.querySelector('#ideaParent');
+
+var savedIdeas = [];
+
+
+saveButton.addEventListener('click', createIdeaCard);
+window.addEventListener('load', checkInputFields);
+titleInput.addEventListener('keyup', checkInputFields);
+bodyInput.addEventListener('keyup', checkInputFields);
+
+
+function createIdeaCard() {
+  if (titleInput.value !== "" && bodyInput.value !== "") {
+    var newIdea = new Idea();
+    newIdea = createNewIdea();
```

Comment:
Line 17 currently declares a new variable called `newIdea` and assigns it a new instance of the Idea class, where the title is undefined and the body is undefined. Line 18 invokes the `createNewIdea` function that also creates a new idea and returns it, where the title and body come from the title and body inputs. Is it necessary to create an empty class first to then overwrite it with the intended idea?